IMP: add rarefy and rarefaction definition to glossary (#571)
qiime2/docs: source/glossary.rst

   qzv
     See :term:`visualization`.

   rarefy

>      A technique to minimize uneven sampling depth by subsampling without 
>      replacement so that all :term: `samples <sample>` have the same sampling

Reviewer: ```suggestion
     A technique to remove uneven sampling depth by subsampling without 
     replacement so that all :term: `samples <sample>` have the same sampling
```

>      depth. As decribed in `Waste not, want not: 
>      Revisiting the analysis that called into question the practice of 
>      rarefaction
>      <https://www.biorxiv.org/content/10.1101/2023.06.23.546312v1.full>` by
>      Patrick Schloss.

Reviewer: ```suggestion
     depth. See `Schloss (2024) <https://doi.org/10.1128/msphere.00355-23>` for additional information. 
```

Shortening this for readability, and also citing the final published paper instead of the preprint.

     

   rarefaction

>      An iterative technique to minimize uneven sampling depth. First, :term: 
>      `rarefy` a feature table. Second, use the table to
>      run diversity metrics. Third, rarefy and run diversity
>      metrics for multiple iterations (typically 100 or 1000) and lastly,

Reviewer: ```suggestion
     An iterative technique to minimize effects of controlling for uneven 
     sampling depth where a feature table is :term:`rarefied<rarefy>` 
     multiple times (typically 100 or 1000), and diversity 
     metrics are applied to each rarefied table and subsequently used to
```

>      compute summary statistics. Also decribed in `Waste not, want not: 
>      Revisiting the analysis that called into question the practice of 
>      rarefaction
>      <https://www.biorxiv.org/content/10.1101/2023.06.23.546312v1.full>` by
>      Patrick Schloss.

Reviewer: ```suggestion
     compute summary statistics. See `Schloss (2024) <https://doi.org/10.1128/msphere.00355-23>` for additional information. 
```
